egs/sprakbanken_swe/s5/local/dict_prep.sh

```diff
@@ -0,0 +1,25 @@
+#prepares dictionary from NST documentation http://www.nb.no/sbfil/dok/nst_leksdat_se.pdf
+#phone file(s) must be retrieved elsewhere
+
+
+KALDI_ROOT=$(pwd)/../../..
+dir=data/local/dict2
+mkdir -p $dir
+
+if [ ! -f $dir/sv.leksikon.tar.gz ]; then 
+    ( wget http://www.nb.no/sbfil/leksikalske_databaser/leksikon/sv.leksikon.tar.gz --directory-prefix=$dir)
+fi
+ 
```

Comment:
little bit confused about  this -- is the lexicon a different than the one on openslr? I know you added some pronunciations into the openslr one, but shouldn't then this one be replaced by the openslr one?